### PR TITLE
2021-12-26-xlm_roberta_large_token_classifier_hrl_xx

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+</div>
+
+## Results
+
+```bash
++--------------------------+---------+
+|chunk                     |ner_label|
++--------------------------+---------+
+|الرياض                    |PER      |
+|فيصل بن بندر بن عبد العزيز|LOC      |
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md:102]: This is wrong, that LOC is a Person (Faisal bin Bandar bin Abdulaziz). Also it's bad indented. Probably we need to chose another example and fix indentation.

File: docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+|Edition:|Official|
+|Input Labels:|[sentence, token]|
+|Output Labels:|[ner]|
+|Language:|xx|
+|Size:|1.8 GB|
+|Case sensitive:|true|
+|Max sentense length:|256|
+
+## Data Source
+
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md:124]: Can you include this as well? I think it's super relevant information.

Language | Dataset
-- | --
Arabic | ANERcorp
German | conll 2003
English | conll 2003
Spanish | conll 2002
French | Europeana Newspapers
Italian | Italian I-CAB
Latvian | Latvian NER
Dutch | conll 2002
Portuguese | Paramopama + Second Harem
Chinese | MSRA

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can you take a look at the requested changes? Thank you!

File: docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md:124]: @josejuanmartinez 
we took this model from hugging face, you know. I think is not ethical to mentioned these data source as if we trained the model. We already show the link on the page to these source.

File: docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-26-xlm_roberta_large_token_classifier_hrl_xx.md]: @josejuanmartinez 

- The languages without latin-alphabet like Japanese, Chinese, Arabic, Korean etc have this issue. The indentation may appear like this on the page. I fixed it manually and this is best as far as I can do 😄 
- I took the text from Arabic wiki and I corrected the entities of result section

**Review by josejuanmartinez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**